app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -112,10 +112,10 @@ def create
         @collection.collection_type_gid = params[:collection_type_gid].presence || default_collection_type.to_global_id
         @collection.attributes = collection_params.except(:members, :parent_id, :collection_type_gid)
         @collection.apply_depositor_metadata(current_user.user_key)
-        add_members_to_collection unless batch.empty?
         @collection.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE unless @collection.discoverable?
         if @collection.save
           after_create
+          add_members_to_collection unless batch.empty?
```

Comment:
The collection has to be saved before adding members, otherwise there isn't an id for the new collection to add to the member's `member_of_collection_ids`.

Reply:
is this a new issue, or a fix for a standing bug?